using-timescaledb/telemetry.md

```diff
@@ -99,25 +110,58 @@ Alternatively, in a `psql` console, run:
 ALTER [SYSTEM | DATABASE | USER] { *db_name* | *role_specification* } SET timescaledb.telemetry_level=off
 ```
 
-If `ALTER DATABASE` is run, then this will disable telemetry for the specified
-database, but not for other databases in the instance. If `ALTER SYSTEM` is
-run, this will disable telemetry for the entire instance.
-Note that superuser privileges are necessary to run `ALTER SYSTEM`.
+If `ALTER DATABASE` is run, then this will turn down the telemetry
+report level for the specified database, but not for other databases
+in the instance. If `ALTER SYSTEM` is run, this will turn down the
+telemetry report level for the entire instance.  Note that superuser
+privileges are necessary to run `ALTER SYSTEM`.
+
+To remove the telemetry job for a database, connect to the database
+using `psql` and execute:
+
+```sql
+DELETE FROM _timescaledb_config.bgw_job
+WHERE job_type = 'telemetry_and_version_check_if_enabled'
+```
+
+>:TIP: To ensure that newly created databases do not use telemetry,
+>remove the telemetry job from the `template1` database.
 
 After running the desired command, reload the new server configuration in order
 for the configuration changes to take effect.
 
-If at a later time you wish to re-enable version checking and telemetry, either
-include the following line in `postgresql.conf`:
+## Enabling Telemetry
+
+If at a later time you wish to re-enable version checking and
+telemetry, you need to both turn up the telemetry reporting level and
+add a telemetry job.
+
+To turn up the telemetry reporting level, either include the following
+line in `postgresql.conf`:
 
 ```
 timescaledb.telemetry_level=basic
 ```
 
-or run the following command in psql:
+or run the following command in `psql`:
 
 ```
 ALTER [SYSTEM | DATABASE | USER] { *db_name* | *role_specification* } SET timescaledb.telemetry_level=basic
 ```
 
+To add a telemetry job, you need to insert it into the `bgw_job` table:
+
+```sql
+INSERT INTO _timescaledb_config.bgw_job(id, application_name, job_type, schedule_interval, max_runtime_retries, retry_period) VALUES
+(1, 'Telemetry Reporter', 'telemetry_and_version_check_if_enabled', INTERVAL '24h', INTERVAL '100s', -1, INTERVAL '1h')
+```
+
+You can of course pick any interval you like, but these are the
+defaults for the telemetry job.
+
+>:WARNING: It is important that `job_type` and the `id` is set to these values.
+
+>:TIP: To ensure that newly created databases use telemetry, add the
```

Comment:
I think this should be removed, it also only works if you installed timescaledb in template1 which has up- and downsides. Background workers are disabled for template-databases.

Reply:
The docker image install `timescaledb` by default in the `template1` database, so if the user want to ensure that newly created databases use, or do not use, telemetry, you need to tweak the contents of `template1`.

Reply:
Removed the mention instead.